feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -24,6 +25,7 @@ import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
+@Suppress("TooManyFunctions", "UnusedPrivateMember")
```

Comment:
I suppose you have fixed "UnusedPrivateMember" problem already (all the private members are used as I see), consider removing it to avoid confusion.